spec/services/ccms/submitters/obtain_document_id_service_spec.rb

```diff
@@ -76,9 +76,9 @@ module Submitters # rubocop:disable Metrics/ModuleLength
           end
 
           it 'only populates the documents array with submittable documents' do
-            expect(legal_aid_application.attachments.map(&:attachment_type)).to eq all_attachment_types
+            expect(legal_aid_application.attachments.map(&:attachment_type).sort).to eq all_attachment_types.sort
             subject.call
-            expect(submission.submission_documents.map(&:document_type)).to eq submittable_document_types
+            expect(submission.submission_documents.map(&:document_type).sort).to eq submittable_document_types.sort
```

Comment:
An alternative would be 
```ruby 
expect(submission.submission_documents.map(&:document_type)).to match_array submittable_document_types
```
That ensures the values in the array match in a non-deterministic sequence

Reply:
Thanks. I knew there would be a method to do it properly 😁 I'll push an update.